linodecli/helpers.py

```diff
@@ -19,17 +19,23 @@
 
 
 def handle_url_overrides(
-    url: str, host: str = None, version: str = None, scheme: str = None
+    url: str,
+    host: str = None,
+    version: str = None,
+    scheme: str = None,
+    override_path=False,
```

Comment:
```suggestion
    host: Optional[str] = None,
    version: Optional[str] = None,
    scheme: Optional[str] = None,
    override_path: bool = False,
```

`str` type needs `Optional[]` to be None-able, and override_path would need a `bool` type to be consistent with other parameters.

Reply:
Nice catch! Will get this fixed shortly